pages/algorithms/correct-autocomplete-field.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Correct autocomplete field
+key: correct-autocomplete-field
+---
+
+Any field name listed in the autocomplete fields table from the HTML 5.2 specification:
+[https://www.w3.org/TR/html52/sec-forms.html#inappropriate-for-the-control](https://www.w3.org/TR/html52/sec-forms.html#inappropriate-for-the-control)
```

Comment:
Don't you prefer linking to https://www.w3.org/TR/html52/sec-forms.html#autofill-field ?